.github/workflows/gh-pages-deploy.yml

```diff
@@ -42,6 +42,9 @@ jobs:
 
       - name: Run deploy script
         working-directory: ./site
+        env:
+          BUILD_ARGS: ${{ github.event.client_payload.build_args }}
```

Comment:
Does this work when the build_args aren't defined?

Reply:
Yes, it will just pass an empty string and build everything like a regular push or manual scrape dispatch.